poll_suggestion/models.py

```diff
@@ -0,0 +1,9 @@
+from django.db import models
+
+
+class poll_suggestion(models.Model):
+    time = models.TimeField()
+    poll_id = models.CharField(max_length=20)
```

Comment:
IMHO the poll should be linked to it like:
```suggestion
    poll_id = models.ForeignKey(Poll, on_delete=models.CASCADE)
```

Reply:
done.